undefined_syms.txt

```diff
@@ -384,7 +384,6 @@ D_040128BC = 0x040128BC;
 D_04014560 = 0x04014560;
 D_04015DB0 = 0x04015DB0;
 D_04015FA0 = 0x04015FA0;
-D_04016360 = 0x04016360;
```

Comment:
Generally it's better to retain these in case other code needs them; we can wipe them all when gameplay_keep is completely identified.